File: GFG-DSA/Java/Mathematics/countOfDigit.java
// count of no. of digits in a number
import java.util.Scanner;

class CountOfDigit {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number:");
        int number = sc.nextInt();
        int count = 0;
        while(number > 0){
            number = number / 10; //123 
            count++; // 3
        }
        System.out.println(count);
    
        sc.close();
    }
}

File: GFG-DSA/Python/Sorting_algo/countofdigit.py
#count of digits in a number



def sumofdigit(n):
    sum = 0
    while n > 0:
        sum += n % 10
        n = n // 10
    return sum

n = int(input("Enter a number: "))  # 3450 12 3

while(n>10):
    n = sumofdigit(n)

print("The number of digits in the number is: ", n)













# def genericroot(n):
#     x = 1+((n-1)%9)
#     return x

# ans = genericroot(int(input("Enter a number: ")))
# print("The root of the number is: ",ans)
